docs/source/index.rst

```diff
@@ -158,3 +158,10 @@ Documentation
    reference/cli
    reference/api
    reference/config
+
+.. toctree::
```

Comment:
How about putting this under User Guides? 

(Not feeling strongly): docs' nav bar is getting a bit long; wdyt about making "Developer Guides" a dropdown? e.g., on click, show CONTRIBUTING / new-cloud.

Reply:
Sounds good. Moved.
For the dropdown. It seems non-trivial to make a single header dropdown. I will probably keep the current expanded version for now.

Reply:
Sounds good. We could look into how to restructure the grouping of different sections; currently they feel a bit too long.